README.md

```diff
@@ -119,14 +119,15 @@ Example:
  curl -vvv "127.0.0.1:8080/api/codex/v1/download/<Cid of the content>" --output <name of output file>
  ```
 
-### `/api/codex/v1/upload`
+### `/api/codex/v1/upload[?chunk=N]`
 
 Upload a file, upon success returns the `Cid` of the uploaded file.
+The optional 'chunk' parameter specifies Manifest.blockSize.
 
 Example:
 
 ```bash
-curl -vvv -H "content-type: application/octet-stream" -H Expect: -T "127.0.0.1:8080/api/codex/v1/upload" -X POST "<path to file>"
+curl -vvv -H "content-type: application/octet-stream" -H Expect: -T "127.0.0.1:8080/api/codex/v1/upload?chunk=65536" -X POST "<path to file>"
 ```
 
```

Comment:
Please make sure that my English is correct here. Also, you may have better wording for this, or have a better name instead of "chunk" for the parameter.

Reply:
`chunk` should be fine.

Reply:
Please check entire changes in README, GitHub GUI shows only few last lines before the comment